tensorflow/tensorflow-engine/src/main/java/ai/djl/tensorflow/engine/TfNDArray.java

```diff
@@ -192,7 +163,7 @@ public NDArray stopGradient() {
                     "DataType mismatch, Required double" + " Actual " + getDataType());
         }
         double[] result = new double[(int) getShape().size()];
-        try (Tensor tensor = operand.asTensor()) {
+        try (Tensor tensor = getHandle().asTensor()) {
```

Comment:
We are still using Tensor, can we bypass Tensor and use TF_Tensor directly?

Reply:
Sure, that's why I am doing now. This is just the first part of huge PR